tasks/openshift-ipi-installer/0.2/openshift-ipi-install.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: openshift-ipi-install-v0-2
+spec:
+  params:
+  - name: IMAGE
+    description: |-
+      Container image to run all the tasks in. The image must have 'openshift-installer' on the $PATH.
+      eg. quay.cor00005-2.cna.ukcloud.com/openshift/openshift-installer
+    type: string
+  - name: OPERATION
+    description: Installation operation to run. Valid options are 'create' or 'destroy'.
+    type: string
+
+  steps:
+  - name: display-version
+    image: $(params.IMAGE)
+    command:
+    - openshift-install
+    - version
+
+  - name: copy-files-from-configmap-for-installer
+    image: $(params.IMAGE)
+    script: cp $(workspaces.config.path)/* $(workspaces.persistent.path)
+
+  - name: cat-file
+    image: $(params.IMAGE)
+    command:
+    - cat
+    - $(workspaces.persistent.path)/install-config.yaml
```

Comment:
Just checking, is there anything potentially sensitive in the install-config we might not want in the pipeline log?

Reply:
Not really it's the cluster configuration so it's quite useful to see what's being deployed and it's passed in with a configmap too so not really sensitive.

Reply:
Ok. Just checking :)